### ISSUE TITLE
tabler-ui package has been renamed and deprecated.

### ISSUE THREAD

**Issue by siburny** [GladysAssistant/Gladys#1005]
`tabler-ui` project has been renamed to `@tabler/core`. This is the messages appearing at top of the page: https://www.npmjs.com/package/tabler-ui

**Comment by atrovato** [comment]
I tried to change the npm depedency, class names changed between our version and this one.
So instead of upgrading tabler version, we may remove this and use boostrap CSS fully.

**Comment by Pierre-Gilles** [comment]
@atrovato : mm let's talk about this, I don't think we want to leave tabler, it's our whole product identity.

Yes, classes changed between the version of tabler we use (from 2 years ago), and the version they deployed right now. I don't see any urgency in switching.

**Comment by siburny** [comment]
It does look like it's a going to be a big project. It can be done once `tabler` release the final version.

**Comment by Pierre-Gilles** [comment]
Yes agree! While they move fast, there is no urgency in trying to switch, it'll be hard to follow their pace (and to understand how to migrate without a documentation)

Let's see when they release the 1.0.0 :)

**Comment by Pierre-Gilles** [comment]
tabler is still not in 1.0.0.

I'm closing this issue because I don't see the point of keeping it open :) We'll see what we do if tabler reach 1.0.0 someday
